--- get_iplayer	2026-04-12 18:50:15
+++ get_iplayer	2026-03-31 09:38:51
@@ -1035,7 +1035,8 @@
 		for my $ep ( @match_list ) {
 			my $ep_desc = " - $ep->{desc}" if $opt->{long} && $ep->{desc};
 			my $ep_type = "$ep->{type}, " if $show_type;
-			main::logger "${ep_type}$ep->{name} - $ep->{episode}, $ep->{channel}, $ep->{pid}${ep_desc}\n";
+            # GIA: Use | as separator for easier parsing
+			main::logger "${ep_type}$ep->{name} - $ep->{episode}|$ep->{channel}|$ep->{pid}${ep_desc}\n";
 		}
 	}
 	main::logger "INFO: ".(scalar @match_list)." total programmes\n";
@@ -6953,6 +6954,7 @@
 		return 'stop';
 	}
 
+    main::logger "INFO: Wrote file $prog->{filename}\n";
 	return 0;
 }
 
@@ -8030,10 +8032,14 @@
 			'p00fzl7g' => 'BBC Radio 5 live', # 5live/programmes/schedules
 			'p00fzl7h' => 'BBC Radio 5 live sports extra', # 5livesportsextra/programmes/schedules
 			'p00fzl65' => 'BBC Radio 6 Music', # 6music/programmes/schedules
+			'p0nkl37k' => 'BBC Radio 6 Indie Forever', # sounds/schedules/bbc_radio_six_indie_forever
 			'p00fzl68' => 'BBC Asian Network', # asiannetwork/programmes/schedules
 			'p00fzl86' => 'BBC Radio 1', # radio1/programmes/schedules
+			'p0hyc2r0' => 'BBC Radio 1 Anthems', # sounds/schedules/bbc_radio_one_anthems
+			'p080kbtk' => 'BBC Radio 1 Dance', # sounds/schedules/bbc_radio_one_dance
 			'p00fzl8v' => 'BBC Radio 2', # radio2/programmes/schedules
 			'p00fzl8t' => 'BBC Radio 3', # radio3/programmes/schedules
+			'p0hyc31m' => 'BBC Radio 3 Unwind', # sounds/schedules/bbc_radio_three_unwind
 			'p00fzl7j' => 'BBC Radio 4', # radio4/programmes/schedules/fm
 			'p00fzl7l' => 'BBC Radio 4 Extra', # radio4extra/programmes/schedules
 			'p02zbmb3' => 'BBC World Service', # worldserviceradio/programmes/schedules/uk
@@ -8797,6 +8803,8 @@
 		) if $opt->{verbose};
 	}
 
+    main::logger "INFO: Finished downloading to file: $file_tmp\n";
+
 	return 0;
 }
 
