--- get_iplayer	2026-04-12 18:50:15
+++ get_iplayer	2026-03-31 09:38:51
@@ -1035,7 +1035,8 @@
 		for my $ep ( @match_list ) {
 			my $ep_desc = " - $ep->{desc}" if $opt->{long} && $ep->{desc};
 			my $ep_type = "$ep->{type}, " if $show_type;
-			main::logger "${ep_type}$ep->{name} - $ep->{episode}, $ep->{channel}, $ep->{pid}${ep_desc}\n";
+            # GIA: Use | as separator for easier parsing
+			main::logger "${ep_type}$ep->{name} - $ep->{episode}|$ep->{channel}|$ep->{pid}${ep_desc}\n";
 		}
 	}
 	main::logger "INFO: ".(scalar @match_list)." total programmes\n";
@@ -6953,6 +6954,7 @@
 		return 'stop';
 	}

+    main::logger "INFO: Wrote file $prog->{filename}\n";
 	return 0;
 }

@@ -8796,6 +8807,8 @@
 			$prev_sequence - $start_sequence + 1
 		) if $opt->{verbose};
 	}
+
+    main::logger "INFO: Finished downloading to file: $file_tmp\n";

 	return 0;
 }
